### PR TITLE
Update of townhall schedule for the next quarter

### PR DESCRIPTION
**Pull request by nagarjunakanamarlapudi**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ![DataHub](docs/imgs/datahub-logo.png)
 
-> 📣 Next DataHub town hall meeting on Oct 30th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/)) 
+> 📣 Next DataHub town hall meeting on November 06th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/))
 > - [Signup sheet & questions](https://docs.google.com/spreadsheets/d/1hCTFQZnhYHAPa-DeIfyye4MlwmrY7GF4hBds5pTZJYM)
 > - VC link (**NEW** we're using zoom!): https://linkedin.zoom.us/j/95617940722
```
Comment (mars-lan) [inline README.md:24]: Let's also drop the "new" part since it's not that new any more :P

**Review by mars-lan — APPROVED** [review]
Otherwise LGTM

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 ![DataHub](docs/imgs/datahub-logo.png)
 
-> 📣 Next DataHub town hall meeting on Oct 30th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/)) 
+> 📣 Next DataHub town hall meeting on November 06th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/))
```
Comment (jplaisted) [inline README.md:22]: Just "6th", no leading 0. Ditto below.

American thing? I've not seen "Month Day, Year" where day has a leading zero before.

File: README.md (hunk 1 above)
Comment (nagarjunakanamarlapudi) [reply README.md:24]: Agree 👍

File: README.md (hunk 2 above)
Comment (nagarjunakanamarlapudi) [reply README.md:22]: I have been adding 0s all the way till now  😄

File: docs/townhalls.md [hunk 3]
```diff
@@ -17,35 +17,35 @@ You can join with this link, or see the full invite below. https://linkedin.zoom
...
-John Plaisted is inviting you to a scheduled Zoom meeting.
+Nagarjuna Kanamarlapudi is inviting you to a scheduled Zoom meeting.
 
 Topic: DataHub Town Hall
-Time: Oct 30, 2020 09:00 AM Pacific Time (US and Canada)
+Time: Nov 6, 2020 09:00 AM Pacific Time (US and Canada)
 
 Join Zoom Meeting
-https://linkedin.zoom.us/j/95617940722
+https://linkedin.zoom.us/j/4698262890
```
Comment (mars-lan) [inline docs/townhalls.md:26]: Why changing the ID?

Comment (nagarjunakanamarlapudi) [reply docs/townhalls.md:26]: I am unable to change the meeting time invite of John's meeting.

Comment (mars-lan) [reply docs/townhalls.md:26]: SGTM

**Review by mars-lan — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -19,9 +19,9 @@
 
 ![DataHub](docs/imgs/datahub-logo.png)
 
-> 📣 Next DataHub town hall meeting on Oct 30th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/)) 
+> 📣 Next DataHub town hall meeting on November 6th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/))
 > - [Signup sheet & questions](https://docs.google.com/spreadsheets/d/1hCTFQZnhYHAPa-DeIfyye4MlwmrY7GF4hBds5pTZJYM)
-> - VC link (**NEW** we're using zoom!): https://linkedin.zoom.us/j/95617940722
+> - VC link (we're using zoom!): https://linkedin.zoom.us/j/95617940722
```
Comment (mars-lan) [inline README.md:24]: Also update this?

Comment (nagarjunakanamarlapudi) [reply README.md:24]: Sorry ... my UI didn't show this until I squashed and merged. Will send out another PR

Comment (mars-lan) [reply README.md:24]: One step closer to T-shirt :)

**Pull request merged**